.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2.1
```

Comment:
@z1haze so this file will be able to connect GitHub to CircleCI and any PR that cannot be built or fails with `npm test` wont be able to be merged.  Just needs to be merged first before that can be setup ;)